src/test/ui/pattern/const-pat-ice.stderr

```diff
@@ -11,3 +11,7 @@ note: rustc VERSION running on TARGET
 
 note: compiler flags: FLAGS
 
+query stack during panic:
```

Comment:
We should probably mention that we're just showing a limited slice of the query stack